app/frontend/hooks/resources/use-requirement-template.ts

```diff
@@ -20,17 +20,15 @@ export const useRequirementTemplate = () => {
       return
     }
 
-    if (!requirementTemplate || !requirementTemplate.isFullyLoaded) {
```

Comment:
Why was this removed?

Reply:
I want to get the latest data on every page visit. Right now if the requirement template is fully loaded once, it won't do a refetch. This can lead to stale data